staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -90,18 +90,18 @@ func (o WardleServerOptions) Validate(args []string) error {
 }
 
 func (o *WardleServerOptions) Complete() error {
-	return nil
-}
-
-func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	// register admission plugins
-	banflunder.Register(o.RecommendedOptions.Admission.Plugins)
+	banflunder.Register(o.RecommendedOptions.Admission)
 
 	// TODO have a "real" external address
 	if err := o.RecommendedOptions.SecureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
```

Comment:
@sttts does this call have the info it needs in Complete()?

Reply:
If we fix plugin append I think this can go back to being in Config.